ondemand.osc.edu/apps/myjobs/templates/Basic_AMBER_Job_Owens/basic_amber.sh

```diff
@@ -1,22 +1,61 @@
-#PBS -N 6pti
-#PBS -l nodes=1:ppn=28
-#PBS -l walltime=0:20:00
+# AMBER 16 Example Batch Script for Owens
+#
+#PBS -N jac9999cuda.owens
```

Comment:
Maybe the job name can be something more appropriate? "ondemand/myjobs/basic_amber" for example?

Reply:
actually i wonder what we do for interactive apps => might be nice to have the same job name convention